storageClient/api.go

```diff
@@ -113,6 +114,28 @@ type GetMembershipKeyRequest struct {
 	GroupID uuid.UUID `json:"group_id"`
 }
 
+// ListGroupRecordsRequest wraps values needed for the request to list all records shared with a group
+type ListGroupRecordsRequest struct {
+	GroupID   uuid.UUID `json:"group_id"`
+	NextToken int64
+	Max       int
+}
+
+// ExecutorQueryResponse contains the high level information for the response sent to client by the executor.
+type ExecutorQueryResponse struct {
+	ResultList   []pdsClient.ListedRecord `json:"results"`
+	LastIndex    int                      `json:"last_index"`
+	TotalResults int64                    `json:"total_results"`
+	SearchID     string                   `json:"search_id"`
```

Comment:
Don’t just copy and paste, read each line you’re pasting and ask yourself do you need it / does it need to be updated. This isn’t needed

Reply:
Thought that had to do with elastic search